src/credentials/CHIPCert.h

```diff
@@ -529,6 +530,14 @@ CHIP_ERROR VerifyCertSignature(const ChipCertificateData & cert, const ChipCerti
  */
 CHIP_ERROR ValidateChipRCAC(const ByteSpan & rcac);
 
+/**
+ * Validates a Network (Client) Identity in TLV-encoded form.
+ *
+ * This function parses the certificate, ensures the rigid fields have the values mandated by the
+ * specification, and validates the certificate signature.
+ */
+CHIP_ERROR ValidateChipNI(const ByteSpan & ni);
```

Comment:
- Please rename `ni` to `network_identity_cert` or a similarly readable name.
- Please document the error for invalid format, and for failed signature, and for success of validation.